source/installer.py

```diff
@@ -154,14 +184,6 @@ def removeOldProgramFiles(destPath):
 			else:
 				os.remove(fn)
 
-	# #7546: Remove old version-specific libs

```

Comment:
Is this issue number still relevant? If so we should probably include it.

Reply:
This issue number was totally wrong. Not sure why it was there in the first place. It was pointing to the one about  not supporting XP anymore.